### DISCUSSION TITLE
Exterior/Math/P%26S/6/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#13] [General]
# Exterior/Math/P%26S/6/

https://savia7582.github.io/Exterior/Math/P%26S/6/

**Comment by IIIILing** [comment]
第六讲第三页第十一个题型:求解近似分布的例题年份写成了16-27[ac01]

**Reply by IIIILing** [reply]
还是这道题答案应该是0.16,这里写成了1.6,p>1了(bushi)
